spec/timezone.html

```diff
@@ -30,73 +30,76 @@ <h1>Time Zone Names</h1>
-    </p>
-
-    <emu-clause id="sec-isvalidtimezonename" aoid="IsValidTimeZoneName">
-      <h1>IsValidTimeZoneName ( _timeZone_ )</h1>
 
+    <emu-clause id="sec-isvalidtimezonename" type="abstract operation">
+      <h1>
+        IsValidTimeZoneName (
+          _timeZone_: a String,
+        )
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It returns *true* if _timeZone_ is an ASCII-case-insensitive match for a built-in time zone name and *false* otherwise.</dd>
+      </dl>
       <p>
-        An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the IsValidTimeZoneName abstract operation as specified in the ECMA-402 specification.
-        If an ECMAScript implementation does not include the ECMA-402 API the following specification of the IsValidTimeZoneName abstract operation is used.
-      </p>
-      <p>
-        The abstract operation IsValidTimeZoneName takes argument _timeZone_, a String value. It returns *true* if _timeZone_, converted to upper case, is equal to one of the built-in named time zone names, converted to upper case.
-        It returns *false* otherwise.
+        Once IsValidTimeZoneName(_timeZone_) has returned *true*, for the lifetime of the surrounding agent, IsValidTimeZoneName(_variant_) must return *true* if _variant_ is an ASCII-case-insensitive match for either _timeZone_ or CanonicalizeTimeZoneName(_timeZone_).
       </p>
+
       <p>
-        Once IsValidTimeZoneName has returned *true* for a particular value of _timeZone_, for the lifetime of the surrounding agent, it may no longer return *false* for that value of _timeZone_, or for the value of CanonicalizeTimeZoneName(_timeZone_).
-      </p>
-      <p>The minimum implementation of IsValidTimeZoneName, supporting only the *"UTC"* time zone, performs the following steps when called:
+        For the purposes of this section, a String value _A_ is an ASCII-case-insensitive match for String value _B_ if the String value derived from _A_ by replacing each occurrence of a lowercase ASCII letter code unit (0x0061 through 0x007A, inclusive) with the corresponding uppercase ASCII letter code unit (0x0041 through 0x005A, inclusive) while preserving all other code units is exactly the same sequence of code units as the String value that is derived from _B_ in the same way.
       </p>
-
       <emu-alg>
-        1. Assert: Type(_timeZone_) is String.
-        1. Let _tzText_ be ! StringToCodePoints(_timeZone_).
-        1. Let _tzUpperText_ be the result of toUppercase(_tzText_), according to the Unicode Default Case Conversion algorithm.
-        1. Let _tzUpper_ be ! CodePointsToString(_tzUpperText_).
-        1. If _tzUpper_ and *"UTC"* are the same sequence of code points, return *true*.
+        1. If _timeZone_ is an ASCII-case-insensitive match for *"UTC"*, return *true*.
```

Comment:
Are you sure this can be editorial? It seems to me that this would be normative if there were, as you described in #1919, "a hypothetical non-ASCII string that case-folds to `"UTC"`", but maybe no such string exists?

Reply:
Correct, that's exactly why I believe this to be editorial.